scripts/requirements-run-test.txt

```diff
@@ -14,3 +14,6 @@ cbor>=1.0.0
 
 # use for twister
 psutil
+
+# Artifacts package creation
+bz
```

Comment:
@LukaszMrugala As per https://github.com/zephyrproject-rtos/zephyr/issues/78622, it looks like a wrong package was added here.